apps/trpc-app-e2e-playwright/tests/app.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { Browser, chromium, Page } from 'playwright';
+import {
+  afterAll,
+  afterEach,
+  beforeAll,
+  beforeEach,
+  describe,
+  expect,
+  test,
+} from 'vitest';
+import { NotesPage } from './fixtures/notes.po';
+import { notes } from './fixtures/notes';
+
+type TRPCTestContext = {
```

Comment:
minor (non-blocking): here you use a type but here:
https://github.com/analogjs/analog/pull/378/files#diff-c1c129e7f86ccef66b1a031bacefabfb777383430505dca7fb95713ea01e51e0R1
you use and interface. Shouldn't we stick to one type?

Reply:
adressed